Buildings/Resources/src/ThermalZones/EnergyPlus/C-Sources/BuildingInstantiate.c

```diff
@@ -640,8 +652,33 @@ void generateAndInstantiateBuilding(FMUBuilding* bui){
         bui->time, bui->modelicaNameBuilding, bui->precompiledFMUAbsPat, bui->fmuAbsPat);
     copyBinaryFile(bui->precompiledFMUAbsPat, bui->fmuAbsPat, SpawnFormatError);
   }
-  else
-    generateFMU(bui, modelicaBuildingsJsonFile);
+  else{
+    /* Try to generate FMU with absolute path */
+#ifdef _WIN32 /* Win32 or Win64 */
```

Comment:
@mwetter I think there needs to be an addition, where spawn2 is the instance of Spawn located by the environment variable SPAWNPATH, and then spawn3 becomes what you now have as spawn2, which is the instance of Spawn on the system path.

You have captured this on the architecture page https://lbl-srg.github.io/soep/softwareArchitecture.html#distribution-of-spawn-and-the-modelica-buildings-library, including SPAWNPATH at my request, however I think this PR was generated before we discussed the SPAWNPATH addition. Can you make a code revision that accounts for all three levels of the spawn search?

Reply:
@kbenne : That is on my to-do list. This code change was indeed before the architecture page was updated.

Reply:
This will be done through https://github.com/lbl-srg/modelica-buildings/issues/2759